react/features/video-quality/middleware.js

```diff
@@ -64,6 +74,20 @@ StateListenerRegistry.register(
         }
 
         if (maxReceiverVideoQuality !== newMaxRecvVideoQuality) {
```

Comment:
I think your change should go before this check, so that we don't dispatch an event when `maxReceiverVideoQuality` doesn't actually change

Reply:
so I used that if to not log on every change, maybe I can add some extra check or return